src/main/java/randoop/condition/OperationConditions.java

```diff
@@ -0,0 +1,175 @@
+package randoop.condition;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Represents the collection of all conditions on an operation including pre-, post- and
```

Comment:
There is a class named `randoop/condition/Condition.java`.  It is reasonable for a user to assume that any use of the word "condition" refers to that.  If you want "condition" to refer to something else, then you need to rename that class.  Otherwise, change occurrences of "condition" in text where it is not referring to `Condition`.

Another way to put this is that the Javadoc says:
> Represents the collection of all conditions 

and you agreed that is incorrect and misleading, because your comment itself says:
> it does represent the collection of all preconditions, pre-post-condition pairs, and pre-throws-condition pairs.

(ie, not a **"collection of conditions"**, but a collection of other things like preconditions and pairs).